politeiawww/www.go

```diff
@@ -144,7 +144,17 @@ func RespondWithError(w http.ResponseWriter, r *http.Request, userHttpCode int,
 
 	if pdError, ok := args[0].(www.PDError); ok {
 		pdErrorCode := convertErrorStatusFromPD(pdError.ErrorReply.ErrorCode)
-		if pdErrorCode == www.ErrorStatusInvalid {
+		switch {
+		case pdErrorCode == www.ErrorStatusIsQuiesced:
+			log.Errorf("%v politeiad is quiesced, writes aren't allowed",
```

Comment:
You don't need to log this.  Only the unexpected errors that cause a 500 need to be logged.